conf/write_data_and_rebuild_index_simultaneously.json

```diff
@@ -0,0 +1,196 @@
+            "user": ""
+        },
+        {
+            "host": "",
+            "install_dir": "",
+            "conf_dir": "",
+            "type": "storaged",
+            "user": ""
+        },
+        {
+            "host": "",
+            "install_dir": "",
+            "conf_dir": "",
+            "type": "storaged",
+            "user": ""
+        },
+        {
+            "host": "",
+            "install_dir": "",
+            "conf_dir": "",
+            "type": "storaged",
+            "user": ""
+        }
+    ],
+    "actions" : [
+        {
+            "type": "StartAction",
+            "inst_index": 0,
+            "depends": [2, 3, 4]
+        },
+        {
+            "type": "StartAction",
+            "inst_index": 1,
+            "depends": []
+        },
+        {
+            "type": "StartAction",
+            "inst_index": 2,
+            "depends": [1]
+        },
+        {
+            "type": "StartAction",
+            "inst_index": 3,
+            "depends": [1]
+        },
+        {
+            "type": "StartAction",
+            "inst_index": 4,
+            "depends": [1]
+        },
+        {
+            "type": "WaitAction",
+            "wait_time_ms": 3000,
+            "depends": [0]
+        },
+        {
+            "type": "ClientConnectAction",
+            "depends": [5]
+        },
+        {
+            "type": "CreateSpaceAction",
+            "space_name": "write_data_and_rebuild_index_simultaneously",
+            "replica": 3,
+            "parts": 100,
+            "depends": [6]
+        },
+        {
+            "type": "UseSpaceAction",
+            "space_name": "write_data_and_rebuild_index_simultaneously",
+            "depends": [7]
+        },
+        {
+            "type": "CreateSchemaAction",
+            "name": "circle",
+            "props": [
+                {"name": "nextId", "type": "string"}
+            ],
+            "edge_or_tag": false,
+            "depends": [8]
+        },
+        {
+            "type": "WaitAction",
+            "wait_time_ms": 1000,
+            "depends": [9]
+        },
+        {
+            "type": "WriteCircleAction",
+            "tag": "circle",
+            "col": "nextId",
+            "total_rows": 40000,
+            "row_size": 20,
+            "start_id": 1,
+            "random_value": true,
+            "depends": [10]
+        },
+        {
+            "type": "CreateIndexAction",
+            "schema_name": "circle",
+            "index_name": "circle_rebuild_index",
+            "field": "nextId",
+            "edge_or_tag": false,
+            "string_field": true,
+            "index_len": 20,
+            "depends": [11]
+        },
+        {
+            "type": "WaitAction",
+            "wait_time_ms": 10000,
+            "depends": [12]
+        },
+        {
+            "type": "WriteCircleAction",
+            "tag": "circle",
+            "col": "nextId",
+            "total_rows": 40000,
```

Comment:
Maybe write another circle or a bigger range circle, if we write the same 40000 rows, hard to tell the index is rebuild from previous circle or this one.

Reply:
The data written twice is different.
The first time write random string data 
The second time I wrote the circle data.

Check whether it is a circle data during `loopup`, not  random string data ,
 which verifies that the data of the rebuild index is the second time write .

Reply:
Please refer to the difference between the two WriteCircleActions on line 104 and line 130